### DISCUSSION TITLE
Silently toggle mods when using presets

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#20085] [Ideas] [answered]
https://user-images.githubusercontent.com/40531871/188028682-c6ad2a5d-2371-4f2c-aca5-3b774782c051.mp4

Would fix this issue, I noticed it was oddly too loud when activating/deactivating presets

**Comment by peppy** [comment]
They should select just like other places which make a batch selection (with a slight delay), not silenced, if anything.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20091
